main.py

```diff
@@ -69,10 +72,47 @@ def h_run(*args):
 	bot = halibot.Halibot()
 	bot.start(block=True)
 
+def h_fetch(*args):
```

Comment:
This might serve better if it was pluggable, supporting multiple types of fetching. (Maybe shell out to wget, for example?)